### DISCUSSION TITLE
RuntimeError: The NVIDIA driver on your system is too old (found version 8000).

### DISCUSSION THREAD

**Discussion by nightjojo** [vladmandic/sdnext#2995] [Q&A]
I got the webUI up and running, however the following error occurs when I click generate.
`RuntimeError: The NVIDIA driver on your system is too old (found version 8000). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver.`

It seems that torch cannot recognize the GPU at all("Engine: backend=Backend.DIFFUSERS compute=cpu device=cpu").
Tried reinstalling HIP-SDK5.7, Python and updated SD next, no luck so far.
A little help would be much appreciated.

OS:Windows 10 22H2
CPU:AMD Ryzen 9 5900X
GPU: Radeon RX 6900 XT
Python: 3.10.6
torch: 2.2.1+cu118
Branch:Master
Commit:82973c4
[console log.txt](https://github.com/vladmandic/automatic/files/14718781/console.log.txt)

**Comment by brknsoul** [comment]
Use DDU to remove all traces of an old nvidia driver, and then reinstall the latest AMD drivers.
